src/components/PersonalInformation/index.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { useContext, useState } from "react";
+import DateInput from "../DateInput/index.jsx";
+import TextInput from "../TextInput/index.jsx";
+import styles from "./styles/styles.module.scss";
+import SurveyInformation from "../SurveyInformation/index.jsx";
+
+const PersonalInformation = () => {
+  const [_, setSurveyInfo] = useContext(SurveyInformation);
+  const [name, setName] = useState("");
+  const [family, setFamily] = useState("");
+  const [birthDate, setBirthDate] = useState("");
+
+  function onChange(title, event) {
```

Comment:
`title` is not a best naming. Please come up with a better one.

Reply:
done